lib/include/kframework/avm/avm-initialization.md

```diff
@@ -52,6 +53,21 @@ withing the group, with it's `<txID>`. Transaction IDs will be assigned sequenti
 
 #### Payment Transaction
 
+```k
+  syntax AlgorandCommand ::= "submit" TransactionCell
+  //-------------------------------------------------
+  rule <k> submit <transaction> TXN </transaction> =>
+           #pushTxnBack(<txID> getTxID(<transaction> TXN </transaction>) </txID>)
+           ...
+       </k>
+       <transactions>
+         TXNS =>
+         <transaction> TXN </transaction>
+         TXNS
+       </transactions>
+//       requires notBool (ID in_txns(<transactions> TXNS </transactions>))
```

Comment:
This is not used yet, correct? Is the plan to use this to insert transactions through the pyk code?

Reply:
It is not used now, and I'm already using a different way of preparing the initial configurations with `pyk`. We should get rid of this command, good catch.

We may re-introduce something along these lines if we ever implement a JSON-decoder to support all SDK, not just `py-algorand-sdk`.